unstructured_api_tools/pipelines/templates/pipeline_api.txt

```diff
@@ -112,6 +113,9 @@ async def pipeline_1(request: Request,
     else:
         media_type = content_type
 {% endif %}
+{% if response_schema %}
+    default_response_schema = output_schema or "{{response_schema}}"
```

Comment:
`output_schema` should be able to be passed in like a header in request like `content_type`.